per/management/commands/load_form_components.py

```diff
@@ -0,0 +1,75 @@
+        csv_file_path = os.path.join(command_dir, "../../fixtures/form_components.csv")
+
+        with open(csv_file_path, newline="", encoding="utf-8") as csvfile:
+            reader = list(csv.DictReader(csvfile))
+
+            form_comp_without_sub_comp = [comp for comp in reader if not comp["Component number"] == "14"]
+            form_comp_with_sub_comp = [item for item in reader if item not in form_comp_without_sub_comp]
+
+            component_dict = {}
+            component_numbers = []
+            component_letters = []
+
+            for row in form_comp_without_sub_comp:
+                if row["Component number"] in component_numbers:
+                    if row["Considerations"].strip() == "Urban":
+                        component_dict[row["Component number"]]["urban_description"] = row["Description"]
+                    if row["Considerations"].strip() == "Climate & environmental":
+                        component_dict[row["Component number"]]["climate_description"] = row["Description"]
+                    continue
+
+                component_dict[row["Component number"]] = {}
+                component_dict[row["Component number"]]["component_name"] = row["Component name"]
+                component_dict[row["Component number"]]["component_letter"] = row["Component letter"]
+                component_dict[row["Component number"]]["epidemics_description"] = row["Description"]
+                component_numbers.append(row["Component number"])
+
+            for row in form_comp_with_sub_comp:
+                if row["Component letter"] in component_letters:
+                    if row["Considerations"].strip() == "Urban":
+                        component_dict[row["Component letter"]]["urban_description"] = row["Description"]
+                    if row["Considerations"].strip() == "Climate & environmental":
+                        component_dict[row["Component letter"]]["climate_description"] = row["Description"]
+                    continue
+
+                component_dict[row["Component letter"]] = {}
+                component_dict[row["Component letter"]]["component_name"] = row["Component name"]
+                component_dict[row["Component letter"]]["component_letter"] = row["Component letter"]
+                component_dict[row["Component letter"]]["epidemics_description"] = row["Description"]
+                component_letters.append(row["Component letter"])
+
+            for k, v in component_dict.items():
+                form_component = FormComponent.objects.filter(title__icontains=v["component_name"].strip()).first()
+                form_component.urban_considerations_guidance = format_description(v["urban_description"])
+                form_component.epi_considerations_guidance = format_description(v["epidemics_description"])
+                form_component.climate_environmental_considerations_guidance = format_description(v["climate_description"])
+                form_component.save(
```

Comment:
Instead of individual save, we can use [BulkManager](https://github.com/IFRCGo/go-api/blob/98acb2eb0481d61d47a471af975ebf5e320a7291/main/managers.py#L69).

Sample  usage
```python
bulk_mgr = BulkUpdateManager(
    update_fields=fields_to_updated,
    chunk_size=20,
)

for i in items:
    bulk_mgr.add(obj_with_id_and_changed_values)


bulk_mgr.done()
self.stdout.write(self.style.SUCCESS(f"Updated: {bulk_mgr.summary()}"))
```